computer.rb

```diff
@@ -0,0 +1,19 @@
+# computer.rb
+
+# Player when the user is the not the codemaker
+class Computer
+
+  attr_accessor :code, :guess
+
+  def generate_code
+    generated_code = []
+    4.times{ generated_code << $colors[rand(6)] }
+    @code = generated_code
+  end
+
+  def get_guess
```

Comment:
`generate_code` and `get_guess` are exactly same in `Player` and `Computer`, you could use one method and use for both cases